packages/x-date-pickers/src/internals/hooks/usePicker/usePickerValue.types.ts

```diff
@@ -209,6 +209,10 @@ export type PickerValueUpdateAction<TValue, TError> =
  * Props used to handle the value that are common to all pickers.
  */
 export interface UsePickerValueBaseProps<TValue, TError> {
+  /**
+   * The `name` attribute of the picker.
```

Comment:
WDYT about such prop description @flaviendelangle @noraleonte?
```suggestion
   * The `name` attribute used on the `field` component.
   * It serves as a `shortcut` for applying the `name` attribute on the `input` HTML element.
```

Reply:
Making the description more explicit makes a lot of sense 💯